### ISSUE TITLE
Line chart title always shown

### ISSUE THREAD

**Issue by seanchen2016** [scanny/python-pptx#293]
I found a issue in plotting charts of one single line. Even though I set chart.has_title = False, the title is still shown anyway.

**Comment by scanny** [comment]
Can you post example code that reproduces the problem?

Are you saying that when you add a second line the title no longer shows?

Also please mention your PowerPoint version and platform.

**Comment by seanchen2016** [comment]
I will try to provide example code later. 
When a second line is added, the title is gone. 
I use PowerPoint 2016 and Unix.
There is no such issue when we use python-pptx 0.6.0.

**Comment by scanny** [comment]
Interesting. I've seen this myself on occasion but was never able to recreate it. The clue about series count is a good one.

When you say PowerPoint 2016 and Unix, do you mean LibreOffice? PowerPoint is only available on Windows and Mac as far as I know.

**Comment by scanny** [comment]
Btw, what text appears in the chart title? You haven't specified any I take it, so it must be picking something up from somewhere. What text appears and where do you suspect it's coming from?

**Comment by seanchen2016** [comment]
We run python-pptx on Unix to create pptx files and review files on Windows.
When using chart_data.add_series('Series Name', value_axis data), 'Series Name' will appear.

**Comment by seanchen2016** [comment]
'West' appears in chart title.
Below is the example code:

from pptx.util import Inches
from pptx.chart.data import ChartData
from pptx.enum.chart import XL_CHART_TYPE
from pptx import Presentation

chart_data = ChartData()
chart_data.categories = ['Q1 Sales', 'Q2 Sales', 'Q3 Sales']
chart_data.add_series('West',    (32.2, 28.4, 34.7))

prs = Presentation()
slide = prs.slides.add_slide(prs.slide_layouts[4])
x, y, cx, cy = Inches(2), Inches(2), Inches(6), Inches(4.5)
chart = slide.shapes.add_chart(XL_CHART_TYPE.LINE, x, y, cx, cy, chart_data).chart

chart.has_title = False
chart.has_legend = True
chart.legend.include_in_layout = False
chart.series[0].smooth = True

prs.save('test.pptx')

**Comment by scanny** [comment]
Ah, so it's the series title of the first series. Interesting. I believe that's what I was seeing as well. Okay, thanks @seanchen2016, this gives me enough to go on. I'll take a look a bit later and see what I can find.

**Comment by Danttee** [comment]
This works for me:

from pptx.util import Inches
from pptx.chart.data import ChartData
from pptx.enum.chart import XL_CHART_TYPE
from pptx import Presentation
from pptx.enum.dml import MSO_LINE
import pptx
import abc

chart_data = ChartData()
chart_data.categories = ['Q1 Sales', 'Q2 Sales', 'Q3 Sales']
chart_data.add_series('West',    (32.2, 28.4, 34.7))
#chart_data.add_series('Eest',    (22.2, 18.4, 24.7))

prs = Presentation()
slide = prs.slides.add_slide(prs.slide_layouts[5])
x, y, cx, cy = Inches(2), Inches(2), Inches(6), Inches(4.5)
chart = slide.shapes.add_chart(XL_CHART_TYPE.LINE, x, y, cx, cy, chart_data).chart

chart.has_title = True
chart.chart_title.text_frame.text = 'dadada'
chart.has_legend = True
chart.legend.include_in_layout = False
if len(chart.series)==1:
    chart.chart_title.text_frame.text = ' '
chart.series[0].smooth = True
chart.series[0].format.line.dash_style=MSO_LINE.DASH
prs.save('test8.pptx')

**Comment by scanny** [comment]
Just a remark I just added to a related issue that sheds some light on this apparently purposeful behavior:

Okay, so here's the logic that I think makes sense of this behavior. Just to recap, in a chart (of at least certain types) that has only _one_ series, the name of the first series appears as the title of the chart without you actually asking PowerPoint to do that.

Leaving aside the question of whether this behavior is _justified_, the reason why this might be _reasonable_ is that the _title can substitute for the legend in this single-series case_.

Generally, a legend maps a color or symbol to a series name, so you can tell which series is representing what. In the single-series case, you still need the name (what the values represent) but you no longer need the color-patch because there's only one, or because the series uses a different color for each value (bar, segment) (common with pie and I expect doughnut charts, but also an option with bar charts).

So a useful option in this single-series case is to turn the legend off and let the title explain what the chart is representing (by displaying the one and only series name). Anyway, opinions could differ on whether that's "helping", but at least it makes sense to me now why they might have done this on purpose.

**Comment by scanny** [comment]
Some further notes on this for when we get back to it. This behavior is evident in the PowerPoint UI. Just create a default column chart and delete the 2nd and 3rd series and you'll see it. The "auto-title" appears containing the name of the first series.

In the UI you can delete the chart title, which adds the following element to the XML:

```xml
  <c:chart>
    <c:autoTitleDeleted val="1"/>  <!-- this one -->
    <c:plotArea>
    ...
```

..after which the chart expands vertically to take up the space formerly occupied by the chart title. So apparently this behavior is called "Auto-title" and is very much on purpose.

It appears that if you _really_ want to get rid of the chart title, you need to add this element as well as getting rid of the title element.

**Comment by scanny** [comment]
Should be fixed in release v0.6.13 later today.
